# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLQueueParamTraits.h
# Commit: 4cff94ca9818
# Full Hash: 4cff94ca98189769f21a855520a345c6c4e5c841
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2021-01-15 03:50:53
# Regressor Bug: 1640607
# File Overlap Count: 1
# Description:
#   Bug 1640607 - Send SurfaceDescriptors for GPU blitting for video-to-webgl. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101061
# ==============================================================================

diff -r 044f8b594c71 -r 4cff94ca9818 dom/canvas/WebGLQueueParamTraits.h
--- a/dom/canvas/WebGLQueueParamTraits.h	Thu Jan 14 01:20:12 2021 +0000
+++ b/dom/canvas/WebGLQueueParamTraits.h	Thu Jan 14 01:23:06 2021 +0000
@@ -68,6 +68,8 @@
 };
 template <>
 struct IsTriviallySerializable<webgl::ReadPixelsDesc> : std::true_type {};
+template <>
+struct IsTriviallySerializable<layers::SurfaceDescriptor> : std::true_type {};
 
 template <typename T>
 struct QueueParamTraits<RawBuffer<T>> {